docs/api/python/symbol.md

```diff
@@ -392,6 +392,21 @@ Composite multiple symbols into a new one by an operator.
     argmin
 ```
 
+### Linear Algebra
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    la_gemm
```

Comment:
I think linalg_* looks better, more consistent with numpy.

I guess this is fine then.

BTW are you planning to add the GPU version?